src/process_utility.c

```diff
@@ -472,10 +472,13 @@ process_alter_foreign_server(ProcessUtilityArgs *args)
 {
 	AlterForeignServerStmt *stmt = (AlterForeignServerStmt *) args->parsetree;
 
-	if (block_on_foreign_server(stmt->servername))
+	if (stmt->has_version)
 		ereport(ERROR,
 				(errcode(ERRCODE_FEATURE_NOT_SUPPORTED),
-				 errmsg("alter server not supported on a TimescaleDB data node")));
+				 errmsg("operation not supported"),
+				 errdetail("It is not possible to set a version on a data node.")));
```

Comment:
Reading this message seems like I'm executing the `ALTER SERVER` on the data node and it's not true because actually we're executing it in the access node to change a data node server configuration, what about rephrasing it a bit?

```suggestion
				 errdetail("It is not possible to set a version to a data node server configuration.")));
```

As I'm not a native speaker not sure if my suggestion is good enough. Otherwise looks good to me.

Reply:
Updated the message.